internal/query/access_token.go

```diff
@@ -2,11 +2,11 @@ package query
 
 import (
 	"context"
-	"strings"
 	"time"
 
 	"golang.org/x/text/language"
 
+	"github.com/zitadel/zitadel/internal/command"
```

Comment:
In ZITADEL architecture we don't allow imports between query and command. They have distinct functionality:

- command handles data mutations by pushing events to the eventstore
- query handels reading of state data from the eventstore or projections

The `domain` package is used for shared definitions like types and enums.